platform/viewer/src/components/PreferencesDropdown/PreferencesDropdown.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import { Dropdown, IconButton, Icon, useModal } from '@ohif/ui';
+
+const PreferencesDropdown = () => {
+  const { show } = useModal();
+
+  const showAboutModal = () => {
+    const modalComponent = () => <div>About modal</div>;
+    show({
+      content: modalComponent,
+      title: 'About',
+    });
+  };
+
+  const showPreferencesModal = () => {
+    const modalComponent = () => <div>Preferences modal</div>;
+    show({
+      content: modalComponent,
+      title: 'Preferences',
+    });
+  };
+
+  return (
+    <Dropdown
+      titleElement={
+        <IconButton
+          variant="text"
```

Comment:
I think this code is duplicated in the Header, maybe we should use the preferences modal instead?

Reply:
Yeah, that's what I was talking in that comment: https://github.com/OHIF/Viewers/pull/1803#discussion_r444632898
Let's discuss offline about it